### DISCUSSION TITLE
a "comments" button

### DISCUSSION THREAD

**Discussion by andreyorst** [jocmp/capyreader#548] [Feature Requests]
Some feeds like lobste.rs, hackernews, reddit, etc are based around posting a link, and basically discussing it. Often, you can toggle to "full content" view to see the article, and toggle back to see the comments or the link to them.

In Twine there's a comment button on items from such feeds:

![image](https://github.com/user-attachments/assets/a49977cb-ff85-4301-8355-91ceae10f835)

It's quite useful as a feature on its own, but also kinda awkward, because you have to close the article to get to the button.

If it's possible, it'd be cool if Capy had such button in the top header of the article, and maybe on the item in the list like on these mockup screenshots:

![image](https://github.com/user-attachments/assets/05077d60-8ff1-431b-a305-5449f6b1e5ca)
![image](https://github.com/user-attachments/assets/e90e07c6-93f6-4721-8762-0b91db002201)

**Comment by andreyorst** [comment]
Oh, sorry, perhaps I was supposed to post these (#547, #546, #545) as discussions, not as issues. Didn't realize that the discussions exist (I don't do Github that much)

**Comment by jocmp** [comment]
No problem about discussions! I end up triaging issues all the same. 

Let me investigate how Twine does this. It would be a cool feature to add at some point.
